js/auth.js

```diff
@@ -0,0 +1,50 @@
+"use strict"
+
+const apiURL = 'http://localhost:3000';
+
+async function getAllUsers() {
+    const response = await fetch(apiURL + '/users', {
+        method: "GET",
+        headers: {
+            'Accept': 'application/json'
+        }
+    });
+    if(response.ok === true){
+        const user = await response.json();
+        return user;
+    }
+};
+
+
+/*  User login servise  */ 
+
+document.forms['loginForm'].addEventListener('submit', e => {
+    e.preventDefault();
+    const form = document.forms['loginForm'];
+    const email = form.elements['email'].value;
+    const password = form.elements['password'].value;
+
+    const body = {
+        email: email,
+        password: password
+    }
+
+    login(body);
+    form.reset()
+});
+
+async function login(body){
+    const allUsers = await getAllUsers()
```

Comment:
### IMPORTANT
Can it be updated to get user from the API by email and password instead of returning all users to UI and checking it here? 

The reasons are the following:
1. SECURITY - You do return all emails and passwords, the response can be spied and someone can easily get all of that information 
2. PERFORMANCE - Let's imagine your system has 10000 users, and you have to return all of those, just to check if such exists